articles/azure-monitor/app/opencensus-python.md

```diff
@@ -56,7 +56,11 @@ First you have to create an Application Insights resource, which will generate a
     > [!NOTE]
     > `python -m pip install opencensus-ext-azure` assumes that you have a PATH environment variable set for your Python installation. If you have not configured this, you would need to give the full directory path to where your Python executable is located which would result in a command like: `C:\Users\Administrator\AppData\Local\Programs\Python\Python37-32\python.exe -m pip install opencensus-ext-azure`.
 
-2. First let's generate some trace data locally. In Python IDLE, or your editor of choice, enter the following code:
+2. The SDK utilizes three Azure Monitor exporters to send different types of telemetry to application insights: trace, metrics and logs. Follow the instructions below to see how to send these different types via the three exporters.
```

Comment:
Add note to this link if customer wants to learn about the difference between metrics, traces, logs.
https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-platform